internal/registry/registry.go

```diff
@@ -195,9 +198,9 @@ func pkgInDir(pkgName, dir string) bool {
 	return currentPkg.Name == pkgName || currentPkg.Name+"_test" == pkgName
 }
 
-func parseImportsAliases(pkg *packages.Package) map[string]string {
+func parseImportsAliases(syntaxTree []*ast.File) map[string]string {
```

Comment:
I changed the argument so that we don't pass more into the function than what's needed.